client/site-profiler/components/site-profiler.tsx

```diff
@@ -63,6 +70,28 @@ export default function SiteProfiler( props: Props ) {
 		urlData
 	);
 
+	const url = getValidUrl( routerDomain );
+
+	const {
+		data: basicMetrics,
+		error: errorBasicMetrics,
+		isFetching: isFetchingBasicMetrics,
+	} = useUrlBasicMetricsQuery( url );
+
+	const showBasicMetrics =
+		basicMetrics &&
+		! isFetchingBasicMetrics &&
+		! errorBasicMetrics &&
+		isEnabled( 'site-profiler/metrics' );
+
+	// TODO: Remove this debug statement once we have a better error handling mechanism
+	if ( errorBasicMetrics ) {
+		debug(
+			`Error fetching basic metrics for domain ${ domain }: ${ errorBasicMetrics.message }`,
```

Comment:
```suggestion
			`Error fetching basic metrics for domain ${ domain }: ${ errorBasicMetrics?.message }`,
```

PS: Just being super safe 😄

Reply:
This is not required, it is inside a `if (errorBasicMetrics )`, so it will always be defined

Reply:
Good point, thanks for explaining